cloudconfig.tpl

```diff
@@ -877,8 +877,8 @@ write_files:
         sudo service WowzaStreamingEngine stop
 
         ## Delete old files
-        sudo mv "$wowza_lib_dir/log4j-core-2.13.3.jar" "$home_dir/patch"
-        sudo mv "$wowza_lib_dir/log4j-api-2.13.3.jar" "$home_dir/patch"
```

Comment:
these will stay the same, as Wowza will default install 2.13.3 and the code will overwrite them each time.

Reply:
ahhhh